torch/csrc/autograd/variable.h

```diff
@@ -796,6 +797,40 @@ inline Variable make_variable(
   return Variable();
 }
 
+struct VariableHooks final : at::impl::VariableHooksInterface {
```

Comment:
Needed to move this so that we can define VariableHooks::basic_autograd_not_implemented_fallback in autograd_not_implemented_fallback.cpp